src/main/java/edu/cornell/kfs/concur/ConcurConstants.java

```diff
@@ -48,5 +48,11 @@
     public static final String ACCEPT_SUB_FOLDER_NAME = "accept";
     public static final String REJECT_SUB_FOLDER_NAME = "reject";
     public static final String FORWARD_SLASH = "/";
-    public static final String CONCUR_PROCESSED_FILE_DATE_FORMAT = "yyyy-MM-dd-HHmmss";
+    public static final String CONCUR_DATE_FORMAT = "yyyy-MM-dd";
+    public static final String CONCUR_PROCESSED_FILE_DATE_FORMAT = CONCUR_DATE_FORMAT + "-dd-HHmmss";
```

Comment:
Per our hallway conversation, CONCUR_PROCESSED_FILE_DATE_FORMAT will have "dd" in it twice in succession with this change.